admin_site/system/views.py

```diff
@@ -38,8 +38,10 @@ def set_notification_cookie(response, message, error=False):
         "message": message,
         "type": "success" if not error else "error"
     }
+
     response.set_cookie('bibos-notification',
-            quote(json.dumps(descriptor), safe=''))
+                        quote(json.dumps(descriptor), safe='')
```

Comment:
(What's up with all this random refactoring of unrelated code?)

Reply:
Again I apologize for my impulsive behavior in regards the code formatting

Reply:
Using `pycodestyle` is an excellent idea (`flake8` makes the same check & adds some sanity checks from `pyflakes` too, so it's to be preferred), but maybe a complete PEP-8-compliance should be (or: Should have been) a different pull request. But I support the idea of running the checks.